src/main/kotlin/com/openlattice/users/Auth0SyncService.kt

```diff
@@ -79,25 +96,28 @@ class Auth0SyncService(
         logger.info("Synchronizing enrollments and authentication cache for user ${user.id}")
         val principal = getPrincipal(user)
 
-        processGlobalEnrollments(principal, user)
+        processGlobalEnrollments(mapOf(principal to user))
         processOrganizationEnrollments(principal, user)
 
         syncAuthenticationCache(principal.id)
-        markUser(user.id)
     }
 
-    private fun markUser(userId: String) {
```

Comment:
Are we assuming the users will only be deleted via API and not through Auth0?

Reply:
I think so, although it looks like the expiration field is never actually referenced anywhere other than being updated in `markUser`